src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -46,7 +47,7 @@ public int Execute()
                 Console.WriteLine($"Migrating project {project}..");
                 var projectDirectory = Path.GetDirectoryName(project);
                 var outputDirectory = projectDirectory;
-                var migrationSettings = new MigrationSettings(projectDirectory, outputDirectory, sdkVersion, msBuildTemplate, _xprojFilePath);
+                var migrationSettings = new MigrationSettings(projectDirectory, outputDirectory, sdkVersion, msBuildTemplate.DeepClone(), _xprojFilePath);
```

Comment:
Should every migration settings have the msbuild template cloned? The two I have seen here are doing that. If that's the case, maybe we should move the call to DeepClone inside the MigrationSettings constructor and remove this burden from the caller.